core-bundle/src/Resources/contao/controllers/BackendMain.php

```diff
@@ -219,7 +219,14 @@ protected function output()
 	{
 		$this->Template->setData($this->compileTemplateData($this->Template->getData()));
 
-		return $this->Template->getResponse();
+		$response = $this->Template->getResponse();
+
+		if (Input::get('popup') !== null)
+		{
+			$response->headers->set('Content-Security-Policy', "frame-ancestors 'self'");
```

Comment:
```suggestion
			$response->headers->set('Content-Security-Policy', "frame-ancestors 'self'", false);
```
This way, an existing CSP header would not get overwritten which is what we want I guess?